src/store/transactions/exit/exit.thunks.ts

```diff
@@ -0,0 +1,183 @@
+import { push } from "connected-react-router";
+import { BigNumber } from "ethers";
+import { CoordinatorAPI, Tx, HermezCompressedAmount, TxFees } from "@hermeznetwork/hermezjs";
+import { getPoolTransactions } from "@hermeznetwork/hermezjs/src/tx-pool";
+import { TxType } from "@hermeznetwork/hermezjs/src/enums";
+import { formatEther } from "@ethersproject/units";
+import { getProvider } from "@hermeznetwork/hermezjs/src/providers";
+
+import { AppState, AppDispatch, AppThunk } from "src/store";
+import * as exitActions from "src/store/transactions/exit/exit.actions";
+import { openSnackbar } from "src/store/global/global.actions";
+import { createAccount } from "src/utils/accounts";
+import { getNextBestForger, getNextForgerUrls } from "src/utils/coordinator";
+import theme from "src/styles/theme";
+// domain
+import { Account, FiatExchangeRates, PooledTransaction, Token } from "src/domain/hermez";
+import { ETHER_TOKEN_ID } from "src/constants";
+
+/**
+ * Fetches the account details for an accountIndex in the Hermez API.
+ * @param {string} accountIndex - accountIndex of the account
+ * @returns {void}
+ */
+function fetchHermezAccount(
+  accountIndex: string,
+  poolTransactions: PooledTransaction[],
+  fiatExchangeRates: FiatExchangeRates,
+  preferredCurrency: string
+): AppThunk {
+  return (dispatch: AppDispatch, getState: () => AppState) => {
+    const {
+      global: { wallet, tokensPriceTask },
+    } = getState();
+
+    dispatch(exitActions.loadAccount());
+
+    if (wallet) {
```

Comment:
Is it me or after checking twice for `wallet !== undefined`, turns out we don't need the wallet?

The same problem is on `transfer.thunks.ts:fetchHermezAccount` where we're still dispatching the error `Ethereum wallet is not loaded`